apps/frontend/src/renderer/components/settings/SecretsManager.tsx

```
@@ -0,0 +1,432 @@
+                        )}
+                        <div className="flex items-center gap-3 mt-1">
+                          <p className="text-xs text-muted-foreground">
+                            {group.accounts.length} {group.accounts.length === 1 ? 'account' : 'accounts'}
+                          </p>
+                          <p className="text-xs text-muted-foreground">
+                            Keys: {group.keyIds.join(', ')}
+                          </p>
+                        </div>
+                      </div>
+                    </div>
+                    <div className="flex gap-2" onClick={(e) => e.stopPropagation()}>
```

Biome (2.1.2)

[error] Static Elements should not be interactive.

To add interactivity such as a mouse or key event listener to a static element, give the element an appropriate role value.

(lint/a11y/noStaticElementInteractions)